test/api/liskApi.js

```diff
@@ -695,4 +694,45 @@ describe('Lisk.api()', () => {
 			});
 		});
 	});
+
+
+	describe('#constructRequestData @now', () => {
+		const address = defaultAddress.address;
+		const requestData = {
+			limit: requestDataLimit,
+			offset: requestOffset,
+		};
+		const expectedObject = {
+			address: '18160565574430594874L',
+			limit: requestDataLimit,
+			offset: requestOffset,
+		};
+		const conflictObject = {
+			address: '18160565574430594874',
+			limit: 0,
+			offset: 5,
+		};
+		const resolvedConflictObject = {
+			address: '18160565574430594874',
+			limit: requestDataLimit,
+			offset: requestOffset,
+		};
+
+		it('should construct optional request data for API helper functions', () => {
+			const createObject = privateApi.constructRequestData({ address }, requestData);
+			(createObject).should.be.eql(expectedObject);
+		});
+
+		it('should construct with variable and callback', () => {
```

Comment:
`should recognise when a callback function is passed instead of an options object`